content/updates/2017-07-07-release-notes.md

```diff
@@ -0,0 +1,47 @@
+- [Static IP addresses for communicating with external apps]({{< relref "docs/apps/static-egress.md" >}}): Outbound traffic from cloud.gov now comes from specific IP addresses to help customers open up a connection between cloud.gov and outside data centers.
+- We recommend you upgrade your Cloud Foundry CLI to [the latest version](https://github.com/cloudfoundry/cli/releases) to get the latest bug fixes.
+- We improved the responsiveness of the dashboard to better reflect actions that happened on the backend.
+
+
+### Platform releases
+
+We upgraded the Cloud Foundry deployment to [v264](https://github.com/cloudfoundry/cf-release/releases/tag/v264). This upgrade addresses this security vulnerability: [CVE-2017-4994: Forwarded Headers in UAA](https://www.cloudfoundry.org/cve-2017-4994/).
+
+You should [restage your application](http://cli.cloudfoundry.org/en-US/cf/restage.html) to [incorporate fixes in the base filesystem](https://docs.cloudfoundry.org/devguide/deploy-apps/stacks.html#cli-commands) and ensure you’re running the most recent language version supported by your [buildpack](https://docs.cloudfoundry.org/buildpacks/).
+
+#### Additional upgrades
+- [RootFS cflinuxfs2 1.133.0](https://github.com/cloudfoundry/cflinuxfs2-rootfs-release/releases/tag/v1.133.0), which address vulnerabilities described in these security notices:
+  - [USN-3309-1: Libtasn1 vulnerability](https://www.ubuntu.com/usn/USN-3309-1/)
+  - [USN-3304-1: Sudo vulnerability](https://www.ubuntu.com/usn/USN-3304-1/)
+  - [USN-3212-2: LibTIFF regression](https://www.ubuntu.com/usn/USN-3212-2/)
+  - [USN-3302-1: ImageMagick vulnerabilities](https://www.ubuntu.com/usn/USN-3302-1/)
+- [Diego 1.19.0](https://github.com/cloudfoundry/diego-release/releases/tag/v1.19.0)
+- Stemcell 3312.29
+- Buildpack updates:
+  - [Java v3.17](https://github.com/cloudfoundry/java-buildpack/releases/tag/v3.17)
+  - [Ruby v1.6.40](https://github.com/cloudfoundry/ruby-buildpack/releases/tag/v1.6.40)
+  - [Go v1.8.4](https://github.com/cloudfoundry/go-buildpack/releases/tag/v1.8.4)
+  - [NodeJS v 1.5.36](https://github.com/cloudfoundry/nodejs-buildpack/releases/tag/v1.5.36)
+  - [Python v1.5.19](https://github.com/cloudfoundry/python-buildpack/releases/tag/v1.5.19)
+  - [PHP v4.3.34](https://github.com/cloudfoundry/php-buildpack/releases/tag/v4.3.34)
+  - [Binary v1.0.13](https://github.com/cloudfoundry/binary-buildpack/releases/tag/v1.0.13)
+  - [.NET core v1.0.19](https://github.com/cloudfoundry/dotnet-core-buildpack/releases/tag/v1.0.19)
+
+### See Also
+
+If you’re interested in details about recent dashboard updates, you can also see the dashboard release notes.
```

Comment:
Yeah! That's in [the doc version](https://docs.google.com/document/d/1Km3z2H8aH3wN-j2RguobNUhWdV7Hngu6AxwC6SHLbW8/edit#), guessing the link was just accidentally left out. :)